iocage/lib/JailConfigResolver.py

```diff
@@ -0,0 +1,89 @@
+import helpers
+
+import shutil
+
+
+class JailConfigResolver(list):
+
+    def __init__(self, jail_config, logger=None):
+        list.__init__(self, [])
+        helpers.init_logger(self, logger)
+        self.jail_config = jail_config
+        self.jail_config.update_special_property(
+            "resolver", new_property_handler=self)
+
+    @property
+    def conf_file_path(self):
+        return "/etc/resolv.conf"
+
+    @property
+    def method(self):
+        if self.value == "/etc/resolv.conf":
+            return "copy"
```

Comment:
I'm not sure if I understood this concept. @araujobsd are you saying that I should return an index from a tuple here rather than the actual value "copy"? Can you please make an example?